guides/v2.1/cloud/basic-information/starter-develop-deploy-workflow.md

```diff
@@ -141,18 +116,18 @@ The generated file is located in `app/etc/`:
 * For 2.1.4 and later: `config.local.php`
 * For 2.2 and later: `config.php`
 
-You will generate the file in the Integration environment where you configured Magento. We walk you through the process of generating the file, adding it to your Git branch, and deploying it.
+You generate the file in the Integration environment where you configured Magento. We walk you through the process of generating the file, adding it to your Git branch, and deploying it.
 
 **Important notes** on Configuration Management:
 
 * Any configuration setting included in the file is locked from editing, or read-only, in the deployed environment. This is one reason we recommend using `scd-dump`.
 
-  For example, we will have you install a module for Fastly in your development environment. You will only configure this module in Staging and Production. Using `scd-dump` keeps those default fields editable.
+  For example, when you clone to create an integration branch, you install a module for Fastly in your development environment. However, you configure the Fastly module only in Staging and Production. Using `scd-dump` keeps those default fields editable.
```

Comment:
clone to create?  You clone your project. In starter, I guess you create the Integration branch with master as the base branch? Earlier, we have the Integration branch already existing, so we would likely create a feature branch from integration in order to develop...right?

And However is usually with a semicolon just before.